crates/cubecl-linalg/src/matmul/components/global/args.rs

```diff
@@ -1,5 +1,6 @@
+use crate::tensor::{VirtualTensorOperations, VirtualTensorOperationsExpand};
 use cubecl::prelude::*;
-use cubecl_core as cubecl;
+use cubecl_core::{self as cubecl};
```

Comment:
I don't know why you changed that, but the previous style is more consistent with the rest of the code base.